custom-connectors/Public360/Artifacts/apiDefinition.swagger.json

```diff
@@ -0,0 +1,281 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Public 360",
+    "description": "\"Public 360\u00b0 is a market leading ECM system in the Nordics. The Power Automate Public 360\u00b0 connector enables our users to create flows that archive cases and documents in to Public 360\u00b0 and retrieve cases and documents from Public 360\u00b0.\n\nThe Public 360\u00b0 connector will enable our users to archive more and easier! That includes archiving social media posts, Outlook items or OneDrive documents. We also want to combine our Public 360\u00b0 connector with Power Automate robotic process automation to make even more powerful automation flows!\"",

```

Comment:
Different parts of the product will/will not support the degree sign. To play it safe and avoid scenarios where it isn't parsed correctly, I would recommend not using it and referring to it as done in the title, `Public 360`.